basic/src/test/java/org/jboss/hal/testsuite/test/configuration/undertow/UndertowOperations.java

```diff
@@ -101,6 +101,11 @@ public String createHTTPSListener(String httpServer) throws IOException, Command
         String name = RandomStringUtils.randomAlphanumeric(6);
         String securityRealm = "SecurityRealm_ssl-test";
         Address realmAddress = Address.coreService("management").and("security-realm", securityRealm);
+        if (client.options().isDomain) {
+            realmAddress = Address.host(client.options().defaultHost)
+                .and("core-service", "management")
+                .and("security-realm", securityRealm);
+        }
```

Comment:
Looking at the Creaper code, this should be handled automatically even for core-services, see `org.wildfly.extras.creaper.core.online.AdjustOperationForDomain#adjust(org.jboss.dmr.ModelNode)`

BTW: Any reason for not using prepared command in creaper `org.wildfly.extras.creaper.commands.undertow.AddHttpsSecurityRealm`?